src/swarm/neo/util/StructPacker.d

```diff
@@ -0,0 +1,349 @@
+/*******************************************************************************
+
+    Functions to pack a struct instance and all contained array buffers into a
+    contiguous chunk of memory.
```

Comment:
I would explicitly stress that packed data can't be copied/moved because it contains interior pointers.

Reply:
Mentioned in method doc.

Reply:
Better to also put it into the module doc as it is the main design point of the whole concept, immediately answering "how is it different from serializer" question.

Reply:
:+1: That's a minor difference, I'd say. I'll add something, anyway.

Reply:
Updated.